### PR TITLE
product page of ecommerce app

### PR DESCRIPTION
**Pull request by devAmlan**
This is the product page of the ecommerce app. Please review my PR .

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -16,6 +17,7 @@
   --gray:#8991a5;
   --grey:#DCDCDC;
   --yellow:#FFBD35;
+  --light-yellow:#ffb41fa9;
   --red:#DA1212;
   --shadowcolor:rgba(0,0,0,0.2);
   --shadowcolor1:rgba(60, 64, 67, 0.15);
```
Comment (dev-prabhat) [inline css/main.css]: try to consist to color code either use hex code or rgba

File: pages/product/product.css [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+}
+.rating_filter{
+  text-align:left;
+  margin-top:2rem;
+}
+.filter input{
+  margin-bottom:7px;
+  color: var(--pink);
+}
+.filter label{
```
Comment (dev-prabhat) [inline pages/product/product.css]: avoid using tags to apply styles

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Mentioned some changes

File: css/main.css (hunk 1 above)
Comment (devAmlan) [reply css/main.css]: ok I will change them to hex , thanks prabhat.

File: pages/product/product.css (hunk 2 above)
Comment (devAmlan) [reply pages/product/product.css]: I will change it to class, thanks.

**Comment by bossmonkey98** [comment]
Try adding bottom border to navbar
You need to work on its responsiveness the filter section is over lapping onto products list on changing the browser window size
try using media screen query

**Review by bossmonkey98 — APPROVED** [review]

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Good to go after making those suggested changes

**Comment by devAmlan** [comment]
thanks, @bossmonkey98 for the review I will work on the responsiveness of the filter part.

**Pull request merged**